mediaengine.go

```diff
@@ -411,3 +414,21 @@ type RTPCapabilities struct {
 	Codecs           []RTPCodecCapability
 	HeaderExtensions []RTPHeaderExtensionCapability
 }
+
+func (r *RTPCodec) collectStats(collector *statsReportCollector, codecType CodecType) {
+	collector.Collecting()
+
+	stats := CodecStats{
+		Timestamp:   statsTimestampFrom(time.Now()),
+		Type:        StatsTypeCodec,
+		ID:          "codec",
+		PayloadType: uint32(r.PayloadType),
+		MimeType:    r.MimeType,
+		ClockRate:   r.ClockRate,
+		Channels:    uint32(r.Channels),
```

Comment:
This is defined as a `unsigned short` in the spec but we can use `uint8` in go. Please update this in the `CodecStats` struct too.